dao/mongo.go

```diff
@@ -167,6 +167,22 @@ func (m *MongoService) GetPractitionerResources(transactionID string) ([]models.
 	return insolvencyResource.Data.Practitioners, nil
 }
 
+func (m *MongoService) GetPractitionerResource(practitionerID string, transactionID string) (models.PractitionerResourceDao, error) {
+	practitioners, err := m.GetPractitionerResources(transactionID)
```

Comment:
Would it be better to write a query to retrieve one practitioner using the collection rather than retrieving all the practitioners and iterating through the list to find the one you are looking for?

Reply:
Another case for this would if we had to remove the retrieval of the list method, then we would have to refactor this method as well